sky/templates/kubernetes-ray.yml.j2

```diff
@@ -188,6 +188,9 @@ provider:
         - apiGroups: [ "networking.k8s.io" ]   # Required for exposing services.
           resources: [ "ingressclasses" ]
           verbs: [ "get", "list", "watch" ]
+        - apiGroups: [ "scheduling.k8s.io" ]
+          resources: [ "priorityclasses" ]
+          verbs: [ "list", "create", "patch" ]
```

Comment:
will this get updated correctly? Do we need to update any permissions docs or whatever? (Helm chart??)